generators/server/templates/gradle/_profile_prod.gradle

```diff
@@ -82,6 +83,10 @@ processResources {
     <%_ } _%>
 }
 
+gitProperties {
```

Comment:
Before this line, you need to add the following code. Otherwise, if there is not `.git/` folder, the build will fail.

```
generateGitProperties {
	onlyIf {
		!source.isEmpty()
	}
}
```